update release blog templates (#2052)
OpenLiberty/blogs: templates/beta-release-post.adoc

// // // // // // // //
// Change the RELEASE_SUMMARY to an introductory paragraph. This sentence is really
// important becuase it is supposed to grab the readers attention.  Make sure to keep // the blank lines 
//
// Throughout the doc, replace RELEASE_VERSION with the version number of Open Liberty, eg: 22.0.0.2-beta
// // // // // // // //

> The link:{url-about}[Open Liberty] RELEASE_VERSION includes the following beta features (along with all GA features):

Reviewer: ```suggestion
The link:{url-about}[Open Liberty] RELEASE_VERSION includes all GA features and the following beta features:
```

Author: I think I'm going to keep this as-is, as I'd like the focus to be on the beta features (and the GA portion is more of an 'fyi')
